src/gource.cpp

```diff
@@ -1150,6 +1150,10 @@ void Gource::readLog() {
         stop_position_reached = true;
     }
 
+    if (stop_position_reached && gGourceSettings.file_idle_time_at_end > 0.0f) {
```

Comment:
I think maybe this should use `(is_finished || stop_position_reached)` as otherwise this will only happen if `stop_at_end` is also set.